html-css/week1/Homework 1/index.html

```diff
@@ -0,0 +1,203 @@
+    <section id="projects">
+        <div class="project-container">
+            <div class="project-title">
+                <div class="section_text_p1">
+                    <p>Browse My Recent</p>
+                </div>
+                <div class="title">
+                    <h3>
+                    Projects
+                    </h3>
+                </div>
+            </div>    
+                <div class="project-content">
+                    <div id="project1" class="project-one">
+                        <img src="assets/event2.jpg" alt="Event project">
+                        <div class="project-content">
+                            <h3>An Event Website</h3>
+                            
+                        </div>
+                        <div  class="project-btn">
+                            <button><a href="https://codepen.io/FarzanehAhmadi/pen/rNErrRb">Code Pen</a></button>
+                        </div>
+                    </div>
+                    <div id="project2" class="project-two">
+                        <img src="assets/survey.jpg" alt="survey project">
+                        <div class="project-content">
+                            <h3>Survey</h3>
+                        </div>
+                        <div class="project-btn">
+                            <button><a href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form">Free Code Camp</a></button>
+                        </div>
+                    </div>
+                    <div id="project3" class="project-three">
+                        <img src="assets/payment.jpg" alt="payment project">
+                        <div class="project-content">
+                            <h3>Payment Gateaway</h3>
+                        </div>
+                        <div class="project-btn">
+                            <button><a href="">Github</a></button>
+                        </div>
+                    </div>
+                </div>
+        </div>    
+    </section>
+    <section id="contact">
+        <div class="contact-container">
+          <div class="contact-title">
+            <h2>Contact Me</h2>
+            <p>Feel free to reach out for collaborations or just to say hi!</p>
+          </div>
+          <div class="contact-info-container">
+            <!-- Email Info -->
+            <div class="contact-info">
+              <img src="assets/email.png" alt="Email Icon" class="icon">
+              <p>Email: <a href="mailto:f_ahmadi74@yahoo.com">f_ahmadi74@yahoo.com</a></p>
+            </div>
+            
+            <!-- GitHub Info -->
+            <div class="contact-info">
+              <img src="assets/github.png" alt="GitHub Icon" class="icon">
+              <p>GitHub: <a href="https://github.com/FarzanehAhmadi" target="_blank">FarzanehAhmadi</a></p>
+            </div>
+          </div>
+        </div>
+      </section>
+    <!-- Footer Section -->

```

Comment:
remove all comments

Reply:
Should I remove comments from the CSS as well? I added them for readability.